app/src/main/kotlin/com/mapzen/erasermap/view/DirectionItemClickListener.kt

```diff
@@ -0,0 +1,10 @@
+package com.mapzen.erasermap.view
+
+import com.mapzen.valhalla.Instruction
+
+/**
+ * Used by [DirectionListAdapter] when instantiating items
+ */
+interface DirectionItemClickListener {
+    abstract fun onDirectionItemClicked(position: Int)
```

Comment:
I'm not sure `abstract` is explicitly needed here. It should be implied in an interface.